### DISCUSSION TITLE
Example for eBay uploading site hosted images?

### DISCUSSION THREAD

**Discussion by thesrs02** [hendt/ebay-api#81] [General]
Hi, I can't seem to find any example for uploading a binary image to ebay using this library, can you help?

Great library btw, thanks

**Comment by dantio** [comment]
Uploading binary is still under development since I had no time to test. You can checkout the next branch: https://github.com/hendt/ebay-api/tree/next
and take a look in the examples: https://github.com/hendt/ebay-api/blob/next/examples/traditional/trading.UploadSiteHostedPictures.ts

**Reply by thesrs02** [reply]
Thanks for the quick response!
